package.json

```diff
@@ -15,16 +21,25 @@
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "lab -t 100 -La code",
+    "test": "lab -t 100 -L -c",
```

Comment:
we try to stick to the hapi convention for tests, [see a sample here](https://github.com/hapijs/hapi/blob/master/package.json#L49)